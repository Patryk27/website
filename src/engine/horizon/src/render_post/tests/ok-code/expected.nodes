Elem(
    Spanned {
        inner: Elem {
            name: Spanned {
                inner: "p",
                span: Span {
                    start: 1,
                    end: 2,
                },
            },
            attrs: [],
            children: [
                Text(
                    Spanned {
                        inner: "\n  before code fragments\n",
                        span: Span {
                            start: 3,
                            end: 28,
                        },
                    },
                ),
            ],
        },
        span: Span {
            start: 0,
            end: 32,
        },
    },
)
Text(
    Spanned {
        inner: "\n\n",
        span: Span {
            start: 32,
            end: 34,
        },
    },
)
Elem(
    Spanned {
        inner: Elem {
            name: Spanned {
                inner: "code",
                span: Span {
                    start: 35,
                    end: 39,
                },
            },
            attrs: [],
            children: [
                Text(
                    Spanned {
                        inner: "\n  ",
                        span: Span {
                            start: 40,
                            end: 43,
                        },
                    },
                ),
                Comment(
                    Spanned {
                        inner: "\n    fn main() {\n        println!(\"first fragment\");\n    }\n  ",
                        span: Span {
                            start: 47,
                            end: 109,
                        },
                    },
                ),
                Text(
                    Spanned {
                        inner: "\n",
                        span: Span {
                            start: 111,
                            end: 112,
                        },
                    },
                ),
            ],
        },
        span: Span {
            start: 34,
            end: 119,
        },
    },
)
Text(
    Spanned {
        inner: "\n\n",
        span: Span {
            start: 119,
            end: 121,
        },
    },
)
Elem(
    Spanned {
        inner: Elem {
            name: Spanned {
                inner: "code-title",
                span: Span {
                    start: 122,
                    end: 132,
                },
            },
            attrs: [],
            children: [
                Text(
                    Spanned {
                        inner: "\n  some python code\n",
                        span: Span {
                            start: 133,
                            end: 153,
                        },
                    },
                ),
            ],
        },
        span: Span {
            start: 121,
            end: 166,
        },
    },
)
Text(
    Spanned {
        inner: "\n\n",
        span: Span {
            start: 166,
            end: 168,
        },
    },
)
Elem(
    Spanned {
        inner: Elem {
            name: Spanned {
                inner: "code",
                span: Span {
                    start: 169,
                    end: 173,
                },
            },
            attrs: [
                Attr {
                    name: Spanned {
                        inner: "lang",
                        span: Span {
                            start: 174,
                            end: 178,
                        },
                    },
                    value: Some(
                        Spanned {
                            inner: "python",
                            span: Span {
                                start: 179,
                                end: 187,
                            },
                        },
                    ),
                },
                Attr {
                    name: Spanned {
                        inner: "class",
                        span: Span {
                            start: 188,
                            end: 193,
                        },
                    },
                    value: Some(
                        Spanned {
                            inner: "code-invalid",
                            span: Span {
                                start: 194,
                                end: 208,
                            },
                        },
                    ),
                },
            ],
            children: [
                Text(
                    Spanned {
                        inner: "\n  ",
                        span: Span {
                            start: 209,
                            end: 212,
                        },
                    },
                ),
                Comment(
                    Spanned {
                        inner: "\n  = def main():\n  =\n  =     print \"second fragment\"\n  =\n  ",
                        span: Span {
                            start: 216,
                            end: 276,
                        },
                    },
                ),
                Text(
                    Spanned {
                        inner: "\n",
                        span: Span {
                            start: 278,
                            end: 279,
                        },
                    },
                ),
            ],
        },
        span: Span {
            start: 168,
            end: 286,
        },
    },
)
Text(
    Spanned {
        inner: "\n\n",
        span: Span {
            start: 286,
            end: 288,
        },
    },
)
Elem(
    Spanned {
        inner: Elem {
            name: Spanned {
                inner: "code",
                span: Span {
                    start: 289,
                    end: 293,
                },
            },
            attrs: [],
            children: [
                Text(
                    Spanned {
                        inner: "\n  ",
                        span: Span {
                            start: 294,
                            end: 297,
                        },
                    },
                ),
                Comment(
                    Spanned {
                        inner: "\n    fn main() {\n  =     println!(\"third fragment\");\n    }\n  ",
                        span: Span {
                            start: 301,
                            end: 363,
                        },
                    },
                ),
                Text(
                    Spanned {
                        inner: "\n",
                        span: Span {
                            start: 365,
                            end: 366,
                        },
                    },
                ),
            ],
        },
        span: Span {
            start: 288,
            end: 373,
        },
    },
)
Text(
    Spanned {
        inner: "\n\n",
        span: Span {
            start: 373,
            end: 375,
        },
    },
)
Elem(
    Spanned {
        inner: Elem {
            name: Spanned {
                inner: "p",
                span: Span {
                    start: 376,
                    end: 377,
                },
            },
            attrs: [],
            children: [
                Text(
                    Spanned {
                        inner: "\n  after code fragments\n",
                        span: Span {
                            start: 378,
                            end: 402,
                        },
                    },
                ),
            ],
        },
        span: Span {
            start: 375,
            end: 406,
        },
    },
)
Text(
    Spanned {
        inner: "\n\n",
        span: Span {
            start: 406,
            end: 408,
        },
    },
)
Elem(
    Spanned {
        inner: Elem {
            name: Spanned {
                inner: "p",
                span: Span {
                    start: 409,
                    end: 410,
                },
            },
            attrs: [],
            children: [
                Text(
                    Spanned {
                        inner: "\n  some ",
                        span: Span {
                            start: 411,
                            end: 419,
                        },
                    },
                ),
                Elem(
                    Spanned {
                        inner: Elem {
                            name: Spanned {
                                inner: "code",
                                span: Span {
                                    start: 420,
                                    end: 424,
                                },
                            },
                            attrs: [],
                            children: [
                                Text(
                                    Spanned {
                                        inner: "inline code",
                                        span: Span {
                                            start: 425,
                                            end: 436,
                                        },
                                    },
                                ),
                            ],
                        },
                        span: Span {
                            start: 419,
                            end: 443,
                        },
                    },
                ),
                Text(
                    Spanned {
                        inner: "!\n",
                        span: Span {
                            start: 443,
                            end: 445,
                        },
                    },
                ),
            ],
        },
        span: Span {
            start: 408,
            end: 449,
        },
    },
)
Text(
    Spanned {
        inner: "\n",
        span: Span {
            start: 449,
            end: 450,
        },
    },
)