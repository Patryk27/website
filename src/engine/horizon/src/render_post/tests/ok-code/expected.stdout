<p>
  before code fragments
</p>

<div class="code"><pre><span></span>fn main() {
    println!(&quot;first fragment&quot;);
}
</pre></div>

<div class="code-title">
  some python code
</div>

<div class="code code-invalid"><pre><span></span><span class="hll"><span class="k">def</span><span class="w"> </span><span class="nf">main</span><span class="p">():</span>
</span><span class="hll">
</span><span class="hll">    <span class="nb">print</span> <span class="s2">&quot;second fragment&quot;</span>
</span></pre></div>

<div class="code"><pre><span></span>fn main() {
<span class="hll">    println!(&quot;third fragment&quot;);
</span>}
</pre></div>

<p>
  after code fragments
</p>

<p>
  some <code>inline code</code>!
</p>
